os/mm/mm_heap/mm_getheap.c

```diff
@@ -95,12 +100,11 @@ int mm_get_heapindex(void *mem)
 			/* A valid address from the user heap for this region would have to lie
 			 * between the region's two guard nodes.
 			 */
-			if ((mem > (void *)g_mmheap[heap_idx].mm_heapstart[region]) && (mem < (void *)g_mmheap[heap_idx].mm_heapend[region])) {
+			if (mem < (void *)(USR_HEAP[heap_idx].mm_heapstart + USR_HEAP[heap_idx].mm_heapsize) && mem >= (void *)USR_HEAP[heap_idx].mm_heapstart) {
```

Comment:
This is chaged some days ago.
(https://github.com/Samsung/TizenRT/commit/447944f1ed595f7c5119757d28562f8ea9ce15e2#diff-0b5f7261f9d67c06ac44fb1c7ab14d5e)
Do you have any reason for reverting this?

Reply:
Here, we are just changing g_mmheap to USR_HEAP.
The changes that you mentioned are in master branch and yet to be integrated to protection branch.